src/util/events.js

```diff
@@ -49,12 +56,6 @@ export function getFilteredEvents(events, filters, zipcodes) {
     }
 
     if (filters.zipcode) {
-      // We do not yet have valid zipcodes to filter against,
-      // give it a pass.
-      if (!zipcodes || !zipCodesLength) {
-        return true;
-      }
-
       const zipcodeLatLng = L.latLng(zipcodes[filters.zipcode]);
```

Comment:
The check above (https://github.com/aclu-people-power/map/pull/58/files#diff-45139cda7f30dab38efba36134c072b6R12) now prevents that error, I believe